### DISCUSSION TITLE
Guidance needed for Full finetuning of SDXL

### DISCUSSION THREAD

**Discussion by Anustup900** [huggingface/diffusers#8784] [Q&A]
I am looking to full finetune the base SDXL model, not Dreambooth / LoRa approach. 
So asking for best practices for those who had did this before for : 
- what should be a good Learning rate to keep ?
- choice of scheduler
- choice of optimizers 
- and steps needed per image ?
Looking forward for some guidelines here

**Comment by asomoza** [comment]
The training scrips here are for learning and experimenting, probably not that much people trained a full SDXL model with the sample scripts, specially if you're going to use a big dataset for which the sample training scripts aren't optimized for.

If you want to do serious SDXL training, I recommend [SimpleTuner](https://github.com/bghira/SimpleTuner) which is based in diffusers and you can refer to the [tutorial](https://github.com/bghira/SimpleTuner/blob/main/TUTORIAL.md) for some recommendations.

Even if you're not going to use it, you can use the codebase and the nice explanations to learn what you need.
